src/pages/CONV_Account_Conversion.page

```diff
@@ -227,10 +236,6 @@
             <div class="slds-backdrop" tabindex="-1"></div>
         </div>
 
-        <apex:form >
-            <apex:actionFunction name="convert" action="{!convert}" reRender="pageMessages" oncomplete="afterConvert()"/>
```

Comment:
@dospromptman moving line 231 into the above form was the fix for #2202, because they were in different forms the value was never being passed into the controller.